Github_Tutorial.ino

```diff
@@ -21,12 +21,11 @@ void setup()
 
 void loop() 
 {
-  byte myValue = 0;
+  int myValue = 0;
```

Comment:
Seems legit per the Arduino reference:
### Returns
The analog reading on the pin. Although it is limited to the resolution of the analog to digital converter (0-1023 for 10 bits or 0-4095 for 12 bits). **Data type: int.**